scripts/homestead.rb

```diff
@@ -410,5 +410,41 @@ def Homestead.configure(config, settings)
             s.args = [settings["ip"]]
             s.privileged = false
         end
+
+        if settings.has_key?("backup") && (Vagrant::VERSION >= '2.1.0' || Vagrant.has_plugin('vagrant-triggers'))
+            dirPrefix = '/vagrant/'
+            settings["backup"].each do |backup|
```

Comment:
Instead of `settings["backup"].each` we should do `settings["databases"].each`. We don't want users to have to configure *which* databases to backup. it should back up all specified in settings, and any possible type (Although I'm find limiting to MySQL, MariaDB, and Postgres for the scope of this PR